archetypes/Portfolio/Overview/ConnectWalletBanner/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import * as Styles from './styles';
+
+type Props = {
+    onClick: () => void;
```

Comment:
Might be worth adding some color to this props maybe rename to handleConnect or something